contrib/patterns/_grid-collapse.scss

```diff
@@ -0,0 +1,3 @@
+.grid__column--collapse {
+  @include grid-collapse(3);
```

Comment:
`grid-collapse` overwrites every property except for `float: left` so it would end up as a lot of extra code output